lib/babe/build.go

```diff
@@ -347,16 +348,17 @@ func hasSlotEnded(slot Slot) bool {
 	return time.Since(slotEnd) >= 0
 }
 
-// ExtrinsicsToBody returns scale encoded block body which contains inherent and extrinsic.
+// ExtrinsicsToBody returns scale2 encoded block body which contains inherent and extrinsic.
```

Comment:
scale2?

Reply:
Had named it scale2 while integrating with the current scale library, forgot to remove from the comments will fix that.